bn/lessons/basics/comprehensions.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: page
```

Comment:
Why we do have these files ??? we can only serve files that are translated right ???